### PR TITLE
added max and minpressure

### PR DESCRIPTION
**Pull request by hnil**
-- add max and min pressure.
This can ensure that one limit evaluations outside tables.
Probably one should change defaults on min temperature to 0 C.

### CODE REVIEW

File: opm/models/blackoil/blackoilnewtonmethod.hh [hunk 1]
```diff
@@ -98,6 +102,18 @@ struct TemperatureMin<TypeTag, TTag::NewtonMethod>
...
+struct PressureMax<TypeTag, TTag::NewtonMethod>
+{
+    using type = GetPropType<TypeTag, Scalar>;
+    static constexpr type value = 1e99; //Kelvin
+};
+template<class TypeTag>
+struct PressureMin<TypeTag, TTag::NewtonMethod>
+{
+    using type = GetPropType<TypeTag, Scalar>;
+    static constexpr type value = -1e99; //Kelvin
```
Comment (GitPaean) [inline opm/models/blackoil/blackoilnewtonmethod.hh:114]: `Kelvin` should be `Pascal`?

**Comment by totto82** [comment]
jenkins build this please

File: opm/models/blackoil/blackoilnewtonmethod.hh (hunk 1 above)
Comment (totto82) [reply opm/models/blackoil/blackoilnewtonmethod.hh:114]: I understand the reasoning of setting PressureMin to -1e99 (i.e. default is no clamping), but negative pressure doesn't look right.  Better to use 0?

Comment (atgeirr) [reply opm/models/blackoil/blackoilnewtonmethod.hh:114]: I think using 0 might lead to test failures and behaviour changes in unexpected places. I suggest merging as is, and then changing the default separately.

**Pull request merged**